ui_framework/doc_site/src/views/menu/menu_example.js

```diff
@@ -1,39 +1,57 @@
 import React from 'react';
 
+import { renderToHtml } from '../../services';
+
 import {
   GuideDemo,
   GuidePage,
   GuideSection,
   GuideSectionTypes,
 } from '../../components';
 
-const menuHtml = require('./menu.html');
-const menuContainedHtml = require('./menu_contained.html');
+import Menu from './menu';
+const menuSource = require('!!raw!./menu');
```

Comment:
Not a fan of the `!!raw` loader stuff. I think we should define our loaders for each extension in the webpack config, and have internally defined loaders like this.